Realm/RLMArrayLinkView.mm

```diff
@@ -278,4 +395,18 @@ - (NSUInteger)indexOfObjectWithPredicate:(NSPredicate *)predicate {
     return RLMConvertNotFound(query.find());
 }
 
+- (NSArray *)objectsAtIndexes:(__unused NSIndexSet *)indexes {
```

Comment:
actually implementing this method is just an optimization (array kvo requires that it exist and either return nil or the correct thing)